docs/requisitos/elicitacao/prototipo.md

```diff
@@ -3,3 +3,100 @@
 ## Histórico de Revisão:
 | Data | Versão | Descrição | Autor |
 |---|---|---|---|
+| 03/05/2019 | 0.1 | Inclusão da introdução e referências | Letícia Meneses |
+| 04/05/2019 | 0.2 | Adicionando as imagens do protótipo | Letícia Meneses |
+|05/05/2019 | 0.3 | Explicação do protótipo | Letícia Meneses |
+
+## Introdução
```

Comment:
A More founded description of what is a protoype at modelling context would be better to understand the file. Use the point of view of someone that never developed and whant to know what does it means.